async/kafka/kafka.go

```diff
@@ -112,14 +110,18 @@ func (f *Factory) Create() (async.Consumer, error) {
 	c := &consumer{
 		brokers:     f.brokers,
 		topic:       f.topic,
-		group:       f.group,
-		traceTag:    opentracing.Tag{Key: "group", Value: f.group},
 		cfg:         config,
 		contentType: f.ct,
-		buffer:      0,
+		buffer:      1000,
```

Comment:
Consider adding it to a const. Are we sure about 1k buffer?

Reply:
I suppose it depends on a lot of factors (network, throughput, etc). However, this was the value before the breaking change. We need to investigate the tweaking of this configuration or we could provide an additional functional option for this to the clients.

Reply:
It was carefully calculated taken into account a lot of variables.
I am kidding this is a wild guess...